examples/python/qiskit_integration.py

```diff
@@ -68,23 +68,14 @@ def vqe_solve(op, maxiter, quantum_instance):
 #
 # We can now run the VQE algorithm. In this example we use only 50 iterations, but greater accuracy may be achieved by increasing this number:
 
+vqe_solve(H2_op, 50, qi)
+
 print("VQE result:", vqe_solve(H2_op, 50, qi))
 
-# Another way to improve the accuracy of results is to apply optimisations to the circuit in an attempt to reduce the overall noise. When we construct our qiskit backend, we can pass in a pytket compilation pass as an additional parameter. There is a wide range of options here; in this example we will use `FullPeepholeOptimise`, which is a powerful general-purpose pass.
-#
-# In order to satisfy the gate-set constraints of pytket compilation passes, it is sometimes necessary to get qiskit to "unroll" to a certain gate set first; this we can do by passing a `PassManager` to the `QuantumInstance` constructor, as illustrated below.
-
-from pytket.passes import FullPeepholeOptimise
-from qiskit.transpiler import PassManager
-from qiskit.transpiler.passes import Unroller
-
-qis_backend2 = TketBackend(b_emu, FullPeepholeOptimise())
-qi2 = QuantumInstance(
-    qis_backend2,
-    pass_manager=PassManager(Unroller(["cx", "h", "rx", "ry", "rz"])),
-    shots=8192,
-    wait=0.1,
-)
+# Another way to improve the accuracy of results is to apply optimisations to the circuit in an attempt to reduce the overall noise. When we construct our qiskit backend, we can pass in a pytket compilation pass as an additional parameter. There is a wide range of options here; we recommend the device-specific default compilation pass, provided by each tket backend. This pass will ensure that all the hardware constrains of the device are met. We can enable tket's most agressive optimisation level by setting the parameter `optimisation_level=2`.
```

Comment:
```suggestion
# Another way to improve the accuracy of results is to apply optimisations to the circuit in an attempt to reduce the overall noise. When we construct our qiskit backend, we can pass in a pytket compilation pass as an additional parameter. There is a wide range of options here; we recommend the device-specific default compilation pass, provided by each tket backend. This pass will ensure that all the hardware constraints of the device are met. We can enable tket's most aggressive optimisation level by setting the parameter `optimisation_level=2`.
```

Reply:
Urgh, a day of debugging has made me very tired... Thanks!